gitea-dev/config/app.ini

```
@@ -0,0 +1,99 @@
+APP_NAME = traP Gitea Develop
+RUN_MODE = prod
+WORK_PATH = /data/gitea
+
+[repository]
+ROOT = /data/git/repositories
+DISABLED_REPO_UNITS = repo.wiki
+
+[repository.local]
+LOCAL_COPY_PATH = /data/gitea/tmp/local-repo
+
+[repository.upload]
+TEMP_PATH = /data/gitea/uploads
+
+[server]
+APP_DATA_PATH = /data/gitea
+DOMAIN = git-dev.trapti.tech
+SSH_DOMAIN = git-dev.trapti.tech
+HTTP_PORT = 3000
+ROOT_URL = https://git-dev.trapti.tech/
+DISABLE_SSH = false
+SSH_PORT = 2200
+SSH_LISTEN_PORT = 2200
+LFS_START_SERVER = true
+START_SSH_SERVER = true
+
+[database]
+PATH = /data/gitea/gitea.db
+DB_TYPE = mysql
+HOST = private.kmbk.tokyotech.org:33060
+NAME = service_gitea_dev
+USER = service_gitea_dev
+LOG_SQL = false
+
+[indexer]
+ISSUE_INDEXER_PATH = /data/gitea/indexers/issues.bleve
+
+[session]
+PROVIDER_CONFIG = /data/gitea/sessions
+
+[picture]
+AVATAR_UPLOAD_PATH = /data/gitea/avatars
+REPOSITORY_AVATAR_UPLOAD_PATH = /data/gitea/repo-avatars
+
+[attachment]
+PATH = /data/gitea/attachments
+
+[log]
+MODE = console
+LEVEL = Debug
+ROOT_PATH = /data/gitea/log
+
+[security]
+INSTALL_LOCK = true
+REVERSE_PROXY_LIMIT = 1
+REVERSE_PROXY_TRUSTED_PROXIES = *
+REVERSE_PROXY_AUTHENTICATION_EMAIL = X-Forwarded-User-Email
+REVERSE_PROXY_AUTHENTICATION_USER = X-Forwarded-User
+
+[service]
+DISABLE_REGISTRATION = true
+REQUIRE_SIGNIN_VIEW = true
+ENABLE_REVERSE_PROXY_AUTHENTICATION = false
+ENABLE_REVERSE_PROXY_AUTO_REGISTRATION = true
+NO_REPLY_ADDRESS = trap.jp
+ENABLE_REVERSE_PROXY_EMAIL = true
+DEFAULT_KEEP_EMAIL_PRIVATE = true
+DEFAULT_ORG_MEMBER_VISIBLE = true
+ENABLE_NOTIFY_MAIL = true
+ENABLE_BASIC_AUTHENTICATION = false
+
+[lfs]
+STORAGE_TYPE = minio
+MINIO_ENDPOINT = s3.ap-northeast-1.wasabisys.com
+MINIO_BUCKET = trap-gitea-dev
+MINIO_LOCATION = ap-northeast-1
+MINIO_USE_SSL = true
+MINIO_CHECKSUM_ALGORITHM = md5
```

この辺の設定は`[storage.minio]`セクションにあるべきです。

```suggestion

[storage.minio]
MINIO_ENDPOINT = s3.ap-northeast-1.wasabisys.com
MINIO_BUCKET = trap-gitea-dev
MINIO_LOCATION = ap-northeast-1
MINIO_USE_SSL = true
MINIO_CHECKSUM_ALGORITHM = md5
```